.github/workflows/core-dependency-update.yml

```
@@ -59,4 +59,28 @@ jobs:
           echo "📦 Extracted core version: ${CORE_VERSION}"
           
           # Run the complete core dependency update pipeline
-          node run-pipeline.mjs core-dependency-update "${CORE_VERSION}" 
\ No newline at end of file
+          node run-pipeline.mjs core-dependency-update "${CORE_VERSION}"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)